lms/djangoapps/teams/static/teams/js/spec/views/topic_teams_spec.js

```diff
@@ -83,7 +81,7 @@ define([
         });
 
         it('does not show actions for a user already in a team in the teamset', function() {
-            var options = {myTopicTeamsCollection: TeamSpecHelpers.createMockTeams()};
+            var options = {topicTeams: ['team']};
```

Comment:
I'm a little confused by the type of this variable. Here, it looks like it's an array of string(s), but  on line 101 it looks like a list of 'team objects' and in team_spec_helpers.js:111 it looks like it's a backbone collection